.github/workflows/wheels.yml

```
@@ -179,6 +170,10 @@ jobs:
         with:
           python-version: "3.x"
 
+      - name: Install cibuildwheel
+        run: |
+          & python.exe -m pip install -r .ci/requirements-cibw.txt
```

Do we need this ampersand?
```suggestion
          python.exe -m pip install -r .ci/requirements-cibw.txt
```

---

I don't think it's needed in this case, but PowerShell syntax can be very confusing.